solution_runner/cli.py

```diff
@@ -0,0 +1,43 @@
+import os
+from pathlib import Path
+from typing import Iterator
+
+import click
+
+import consts
+from consts import CliConstants
+from utils import get_module_from_filepath
```

Comment:
```suggestion
from . import consts
from .consts import CliConstants
from .utils import get_module_from_filepath
```
And similarly in all other modules.